src/main/java/org/junitpioneer/jupiter/DefaultLocaleExtension.java

```diff
@@ -67,18 +67,37 @@ private Locale readLocaleFromAnnotation(ExtensionContext context) {
 	}
 
 	private static Locale createLocale(DefaultLocale annotation) {
-		if (!annotation.country().isEmpty() && !annotation.variant().isEmpty()) {
-			return new Locale(annotation.language(), annotation.country(), annotation.variant());
+		if (!annotation.value().isEmpty()) {
+			return createFromLanguageTag(annotation);
 		}
-		else if (!annotation.country().isEmpty()) {
-			return new Locale(annotation.language(), annotation.country());
+		else {
+			return createFromParts(annotation);
 		}
-		else if (!annotation.variant().isEmpty()) {
+	}
+
+	private static Locale createFromLanguageTag(DefaultLocale annotation) {
+		if (!annotation.language().isEmpty() || !annotation.country().isEmpty() || !annotation.variant().isEmpty()) {
 			throw new ExtensionConfigurationException(
-				"@DefaultLocale.country must not be empty when @DefaultLocale.variant is set!");
+				"@DefaultLocale can only be used with language tag if language, country, and variant are not set");
+		}
+		return Locale.forLanguageTag(annotation.value());
+	}
+
+	private static Locale createFromParts(DefaultLocale annotation) {
+		String language = annotation.language();
+		String country = annotation.country();
+		String variant = annotation.variant();
+		if (!language.isEmpty() && !country.isEmpty() && !variant.isEmpty()) {
+			return new Locale(language, country, variant);
+		}
+		else if (!language.isEmpty() && !country.isEmpty()) {
+			return new Locale(language, country);
+		}
+		else if (!language.isEmpty() && variant.isEmpty()) {
+			return new Locale(language);
 		}
 		else {
-			return new Locale(annotation.language());
+			throw new ExtensionConfigurationException("@DefaultLocale not configured correctly");
```

Comment:
```suggestion
			throw new ExtensionConfigurationException(
					"@DefaultLocale not configured correctly. When not using a language tag, specify either"
							+ "language, or language and country, or language and country and variant.");
```

(Coding in GitHub's comment editor - what could possibly go wrong. :joy:)